Replication and Deployment.lvproj

```diff
@@ -423,12 +423,6 @@
 				<Item Name="compatCalcOffset.vi" Type="VI" URL="/&lt;vilib&gt;/_oldvers/_oldvers.llb/compatCalcOffset.vi"/>
 				<Item Name="compatFileDialog.vi" Type="VI" URL="/&lt;vilib&gt;/_oldvers/_oldvers.llb/compatFileDialog.vi"/>
 				<Item Name="General Error Handler Core CORE.vi" Type="VI" URL="/&lt;vilib&gt;/Utility/error.llb/General Error Handler Core CORE.vi"/>
-				<Item Name="nirioFlashWriteBitstream.vi" Type="VI" URL="/&lt;vilib&gt;/LabVIEW Targets/FPGA/RIO/nirioFlashTools.llb/nirioFlashWriteBitstream.vi"/>
-				<Item Name="nirioFlashEraseBitstream.vi" Type="VI" URL="/&lt;vilib&gt;/LabVIEW Targets/FPGA/RIO/nirioFlashTools.llb/nirioFlashEraseBitstream.vi"/>
-				<Item Name="_nirio_device_handleType.ctl" Type="VI" URL="/&lt;vilib&gt;/LabVIEW Targets/FPGA/RIO/nirio_driverPrimitives.llb/_nirio_device_handleType.ctl"/>
-				<Item Name="_nirio_device_close.vi" Type="VI" URL="/&lt;vilib&gt;/LabVIEW Targets/FPGA/RIO/nirio_driverPrimitives.llb/_nirio_device_close.vi"/>
-				<Item Name="niFpgaBitfileGet_All.vi" Type="VI" URL="/&lt;vilib&gt;/rvi/Bitfile/Public/niFpgaBitfileGet_All.vi"/>
-				<Item Name="_nirioDeviceConnect.vi" Type="VI" URL="/&lt;vilib&gt;/LabVIEW Targets/FPGA/RIO/nirioFlashTools.llb/_nirioDeviceConnect.vi"/>
```

Comment:
These dependencies were automatically removed from the .xml when I updated the project setting.

Reply:
In general, that's fine. These dependencies will be recalculated on load anyway. But did you save this project on a machine with LV FPGA installed? It looks like this project references FPGA and so it would be best to save the official copy with FPGA installed. 

Also, have we standardized on a version of LV we are going to use to save updates?

Reply:
I saved the project on a machine with LabVIEW 2014 SP1 and the matching versions of the RT/FPGA modules installed. Since this was the version the project had already been saved in my thought was to leave the version untouched whenever possible.